model_analyzer/result/run_config_measurement.py

```diff
@@ -86,7 +86,7 @@ def from_dict(cls,
             run_config_measurement,
             run_config_measurement_dict['_model_config_measurements'])
 
-        # By default setting all models to have equal weighting
+        # By default set all models to have equal weighting
```

Comment:
Are we going to write out and read in the model weights? Should we?

Reply:
We are not, and we should not. Weights, like constraints and objectives are not stored in the checkpoint and must be specified by the user each time MA is run (else the defaults are used).